modules/unsupported/process-feature/src/test/resources/org/geotools/process/vector/test-data/featuresToGroupWithGroupAllNull.properties

```diff
@@ -0,0 +1,13 @@
+_=the_geom:MultiPolygon:srid=4326,cat:java.lang.Long,group:String,group2:String,numericVal:java.lang.Integer
+features.1=MULTIPOLYGON(((2 2,2 3,3 3,3 2,2 2)), ((3 9,3 12,5 12,5 9,3 9)))|10|group1|secondGroup1|null
```

Comment:
Is it just me, or there is no group with all values at null? I can only see groups with a mix of nulls and non nulls (which needs to be tested for sure, but the title talks about features with null attribute).